pkg/metrics/collector_test.go

```diff
@@ -1,9 +1,10 @@
 package metrics
 
 import (
-	"github.com/aquasecurity/trivy-operator/pkg/apis/aquasecurity/v1alpha1"
 	"strings"
 
+	"github.com/aquasecurity/trivy-operator/pkg/apis/aquasecurity/v1alpha1"
```

Comment:
Nice to fix the wrong placement of this import. I am used to 3 import "groups":

1. Standard Go imports
2. 3. party dependencies
3. Internal dependencies

WDYT? Do you have a standard for organizing imports in this project? Maybe for another PR?